src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4decoder/common/decoder/controllingmetric/QuadMode.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.x4decoder.common.decoder.controllingmetric;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Set of quad mode option
+ *
+ * @author Harry
+ * @since 1.0
+ */
+public enum QuadMode {
+
```

Comment:
Please format the code, adding more space for `NORMAL("Normal", 0)` instead of `NORMAL("Normal",0)`

Reply:
I fixed it and the similar cases